client/src/components/ProfileSettings/EditProfile/EditProfile.tsx

```diff
@@ -1,7 +1,307 @@
-const EditProfile = (): JSX.Element => {
-    return(
-        <h1>This is profile edit panel</h1>
-    );
-}
+import React, { useState } from 'react';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import FormControl from '@material-ui/core/FormControl';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import TextareaAutosize from '@material-ui/core/TextareaAutosize';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import MuiPhoneNumber from 'material-ui-phone-number';
+import axios from 'axios';
 
-export default EditProfile;
\ No newline at end of file
+const theme = {
+  spacing: 8,
+};
+
+const EditProfile = () => {
+  const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+      paper: {
+        marginTop: theme.spacing(8),
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+      },
+      form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(3),
+      },
+      submit: {
+        margin: theme.spacing(3, 0, 2),
+        padding: '0.5rem',
+        width: '10%',
+        color: '#dc004e',
+        border: '1px solid #dc004e',
+      },
+      textbox: {
+        width: '100%',
+      },
+      typography: {
+        fontWeight: 800,
+        fontSize: '12px',
+      },
+      gridcontainer: {
+        padding: theme.spacing(1),
+      },
+      heading: {
+        fontWeight: 550,
+      },
+      phone: {
+        padding: '1px', //TODO add padding or margin later
+      },
+      gender: {
+        fontSize: '10px',
+      },
+    }),
+  );
+  const classes = useStyles();
+  const [phone, setPhone] = useState('');
+  const phoneRegExp =
+    /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/;
+  return (
+    <Formik
+      initialValues={{
+        firstName: '',
+        lastName: '',
+        email: '',
+        birthdate: '',
```

Comment:
This maybe a typo, please update variable name to use camelCase here.

Reply:
Done!

Reply:
Please update this to camelCase - `birthDate`